docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -221,11 +227,6 @@ collectors:
             - key: service.name
               from_attribute: app.label.component
               action: insert
-        attributes/k8s_logs_dataset:
-          actions:
-            - key: data_stream.dataset
-              value: "kubernetes.container_logs"
```

Comment:
remove setting `data_stream.dataset`, so all logs will be stored in default dataset - `generic.otel`

Reply:
I agree with this. Lets remove it